server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -323,6 +323,12 @@ Issues related to authorization may exist just for some other versions of {jdgse
 +
 It is currently required to have different configuration files for the JDG servers on both sites as the Infinispan subsystem does not support replacing site names with expressions. See link:https://issues.redhat.com/browse/WFLY-9458[this issue] for more details.
 
+NOTE: The _PUBLIC_IP_ADDRESS_ below refers to the IP address or hostname, which can be used for your server to bind to. Note that
```

Comment:
`Note that every {jdgserver_name} server and {project_name} server needs to use different address.`

Is it really they need to be a different IP address / hostname, or is it different `IP:port` combination enough? If the latter is the case (different IP address / hostname + port combination is enough), I would probably replace `_PUBLIC_IP_ADDRESS_` with `_SOCKET_ADDRESS_`, so it's clear it's just hostname/IP + port combination, which needs to be unique.

More about [socket address term](https://en.wikipedia.org/wiki/Network_socket#Socket_addresses), namely `In this context, sockets are assumed to be associated with a specific socket address, namely the IP address and a port number for the local node, and there is a corresponding socket address at the foreign node (other node), which itself has an associated socket, used by the foreign process. Associating a socket with a socket address is called binding. ` part of it.

Reply:
For this example, there is _PUBLIC_IP_ADDRESS_ used already and there are no ports involved. Just different virtual hosts like "node1" . I suggest to keep this section as the _PUBLIC_IP_ADDRESS_ is already used in all the example commands below.

Reply:
Ok, fair enough. Thanks